Update Forum Helpers Page JavaScript

Accio1 opened the pull request:
### Resolves:

This resolves #65 and most of #31

### Changes:
-Switch to fetch() instead of XMLRequests
-Add ocular statuses to the list of Forum Helpers page.
-Add the ability to get post counts from ScratchDB, but that seems to be down right now, so it is currently disabled.
-Display the number of curators, managers, and total members on the list of Forum Helpers page.

### Local Tests:

Tested Locally

@jeffalo @leahcimto

Accio1 commented:
Bad bot

leahcimto commented on forumhelpers/FHP.js:
```diff
@@ -1,53 +1,60 @@
+						<a href="https://scratch.mit.edu/users/${name}/"><img src="https://cdn2.scratch.mit.edu/get_image/user/${id}_60x60.png" class="profilePicture" id="${name}pfp" loading="lazy" alt="${name}'s Profile Picture"></a>
+						<p class="profileBio">${bio}</p>
+					</div>
+					<p class="FullOcular" id="${name}Ocular" style="border: 5px solid black; border-radius: 10px; padding: 5px; margin-right:20%; width: fit-content" title="Ocular Status"><span class="ocularStatus" id="${name}Status">Loading Status...</span></p>
```
Could you use External CSS instead of inline?

Accio1 replied:
Right, I was so intent on getting the CSS for that to look right that I forgot to move it to the stylesheet.

leahcimto reviewed (approved):
Just previewed it locally and it looks amazing!

Accio1 commented:
> Just previewed it locally and it looks amazing!

Thanks! I'm going to wait for @jeffalo just to make sure that he also agrees since it is his site that the data is being pulled from.

jeffalo commented:
just checked it out, looks cool.

honestly, you don't need the delay between fetching my-ocular. i can take care of things on my end to make sure im not getting unintentionally (d)dos'd.

FunctionalMetatable reviewed (commented):
I think you can use /v3/forum/user/info instead of /v2/forum/user/info now. (wait am I allowed to review this)

Accio1 commented:
@jeffalo I've removed the delay for fetching Ocular Statuses.
@FunctionalMetatable thanks! I've changed the link and it is working now! I added a half second delay for fetching post counts from ScratchDB so that it doesn't spam their servers.

Since this has been approved, I'm going to go ahead and merge. This page still requires a few small updates, namely making the post count more distinct, but those can be worked on in the future.

leahcimto commented:
Yay! 🎉
